src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -249,6 +249,7 @@ private sealed class InvokeMethod : SourceDelegateMethodSymbol
                     binder, this, syntax.ParameterList, out arglistToken,
                     allowRefOrOut: true,
                     allowThis: false,
+                    shouldPlaceIsConstModifier: false,
```

Comment:
> I think we want "true" here. Otherwise old compiler may convert methods with regular "ref" to delegate types with "ref readonly".

Please add VB test for this scenario.